### ISSUE TITLE
Opening a large design on Windows crashes the app

### ISSUE THREAD

**Issue by edDimensi** [avocode/avocode#915]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.8.0
Source UID: 68799
```
RangeError: Failed to execute 'getImageData' on 'CanvasRenderingContext2D': Out of memory at ImageData creation
    at RangeError (native)
    at e.exports.e.drawImage (file:///C:/Users/Dimen/AppData/Local/avocode/app-2.8.0/resources/app.asar/dist/avocode.min.js:24:5280)
    at file:///C:/Users/Dimen/AppData/Local/avocode/app-2.8.0/resources/app.asar/dist/avocode.min.js:24:3236
    at List.__iterate (C:\Users\Dimen\AppData\Local\avocode\app-2.8.0\resources\app.asar\node_modules\immutable\dist\immutable.js:2208:13)
    at List.forEach (C:\Users\Dimen\AppData\Local\avocode\app-2.8.0\resources\app.asar\node_modules\immutable\dist\immutable.js:4383:19)
    at e.exports.e.createSprites (file:///C:/Users/Dimen/AppData/Local/avocode/app-2.8.0/resources/app.asar/dist/avocode.min.js:24:2834)
    at file:///C:/Users/Dimen/AppData/Local/avocode/app-2.8.0/resources/app.asar/dist/avocode.min.js:24:3164
    at List.__iterate (C:\Users\Dimen\AppData\Local\avocode\app-2.8.0\resources\app.asar\node_mod
```

**Comment by edDimensi** [comment]
Where i can download 2.6 version? I can't work!

**Comment by jakubzitny** [comment]
Hi @edDimensi, thanks for your report.

You can (download Avocode 2.6 for Windows here)[https://media.avocode.com/download/avocode-app/2.7.0/avocode-app-windows-2.7.0.exe].

We'd be happy if you gave us more insight in your problems. Is this `Out of memory` error the only problem you're seeing? You're working under Windows, where we currently have only 32-bit version of the app. Can you tell us what hardware are you using? We're considering right now to release 64-bit Windows app which might solve this partivular problem.

Thanks.

**Comment by edDimensi** [comment]
```
Windows 10 x64 Pro
AMD FX(tm)-4350 Quad-Core Processor, 4200 МГц
8 GB RAM
Geforse 650 Ti
```
The first time I got at least some error. In other cases, the program crashed. Just a white screen and no response. I tried to re-load design and run, but problem is not solved

**Comment by jakubzitny** [comment]
Thanks for the info, we'll try to solve this as soon as possible and we'll let you know here.

I also fixed [the link](https://media.avocode.com/download/avocode-app/2.6.0/avocode-app-windows-2.6.0.exe), it's working now, sorry.

**Comment by edDimensi** [comment]
Thanks

**Comment by vojtatranta** [comment]
Hi @edDimensi  
this is clearly a problem of 32bits version of Avocode.

We backed x64 build of Avocode for you. You can try it out: http://media.avocode.com/download/avocode-app/2.8.0/avocode-app-windows-x64-2.8.0.exe

Opening a design should be OK, because Avocode would be able to allocate more memory.
Note that you will not lose any of your data, but you may experience bad performance first few minutes after login - app needs to download assets and other stuff.
Thank you!

**Comment by jakubzitny** [comment]
Hey, we've just released [Avocode 2.8.1](http://avocode.com/download/) with 64-bit version for Windows. This issue should be fixed in it. Please, check it out and let us know if you have any more problems with it.

Thanks ;)

**Comment by vojtatranta** [comment]
Hi @edDimensi,
here's link for stable Avocode 64bit Windows version: https://manager.avocode.com/download/avocode-app/windows-x64/

It should resolve all memory troubles. Thanks.
